db/migrate/20240111192555_create_posts.rb

```diff
@@ -1,11 +1,11 @@
 class CreatePosts < ActiveRecord::Migration[7.1]
   def change
     create_table :posts do |t|
-      t.references :author, null: false, foreign_key: true
+      t.references :user, null: false, foreign_key: true
```

Comment:
- You have done well so far, but as shown in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png), the column in the post table that references the user table should be named `author`. Please fix this 🙏.

```ruby
     t.references :author, foreign_key: { to_table: 'users' }
```